models/application.ts

```diff
@@ -0,0 +1,91 @@
+import mongoose from 'mongoose';
+
+const Schema = mongoose.Schema;
+
+export const ApplicationSchema = new Schema(
+	{
+		firstName: {
+			type: String,
+			required: true,
+		},
+		preferredName: {
+			type: String,
+		},
+		lastName: {
+			type: String,
+			reqired: true,
+		},
+		gender: {
+			type: String,
+			required: true,
+		},
+		dietaryRestrictions: {
+			type: Array,
+			required: true,
+		},
+		phoneNumber: {
+			type: String,
+			required: true,
+		},
+		dateOfBirth: {
+			type: String,
+			required: true,
+		},
+		school: {
+			type: String,
+			required: true,
+		},
+		major: {
+			type: String,
+			required: true,
+		},
+		graduationYear: {
+			type: String,
+			required: true,
+		},
+		race: {
+			type: Array,
+			required: true,
+		},
+		motivation: {
+			type: Array,
+			required: true,
+		},
+		attendingInPerson: {
```

Comment:
is this a field required by MLH or something from previous years we maintained? if the latter, we should remove it as all hackers need to be in-person